features/scv/jceks-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -24,7 +24,7 @@
     </cm:property-placeholder>
 
     <bean id="jceksSecureCredentialsVault" class="org.opennms.features.scv.jceks.JCEKSSecureCredentialsVault" destroy-method="destroy">
-        <argument value="$[karaf.etc]$[file.separator]scv.jce"/>
+        <argument value="$[karaf.etc]$[file.separator]$[org.opennms.features.scv.keystore.file.name:scv.jce]"/>
```

Comment:
Make sure SCV works fine on Minion as well  ( to validate scv loading from blueprint)